docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc

```diff
@@ -5,42 +5,57 @@
 beta::[]
 
 {kib} {alert-features} include support for {ml} rules, which run scheduled 
-checks on an {anomaly-job} or a group of jobs to detect anomalies with certain 
-conditions. If an anomaly meets the conditions, an alert is created and the 
-associated action is triggered. For example, you can create a rule to check an 
-{anomaly-job} every fifteen minutes for critical anomalies and to notify you in 
-an email. To learn more about {kib} {alert-features}, refer to 
+checks on an {anomaly-job} or a group of jobs to detect anomalies or check the 
+health of the job with certain conditions. If a rule meets the conditions, an 
+alert is created and the associated action is triggered. For example, you can 
+create a rule to check an {anomaly-job} every fifteen minutes for critical 
+anomalies and to notify you in an email. To learn more about {kib} 
+{alert-features}, refer to 
 {kibana-ref}/alerting-getting-started.html#alerting-getting-started[Alerting].
 
+The following {ml} alerts are available:
 
-[[creating-anomaly-alert-rules]]
+* {anomaly-detect-cap} alert: checks if the {anomaly-job} results contain
+  anomalies that match the rule conditions.
+* {anomaly-jobs-cap} health: monitors job health and alerts if an operational 
+  issue occured that may prevent the job from detectiong anomalies. Recommended 
+  to use for jobs with critical importance.
```

Comment:
maybe worth mentioning jobs that have associated alerting rules of `{anomaly-detect-cap}`

Reply:
Further clarified in b25c915.